tests/raw_query/models.py

```diff
@@ -32,6 +32,10 @@ class Coffee(models.Model):
     price = models.DecimalField(max_digits=10, decimal_places=2, default=0)
 
 
+class Tea(models.Model):
+    id = models.AutoField(primary_key=True, db_column='TeA_iD')
```

Comment:
Good catch 🎯, but I think that the main issue here is that `self.columns` (see [[1]](https://github.com/django/django/pull/8511/commits/c4f62648436d3300ea5921cd1ba1a64cb8e5d975#diff-5b0dda5eb9a242c15879dc9cd2121379R1163)) contains converted columns names and `self.model._meta.fields` contains raw db columns, e.g. `TeA_iD`. I'm not sure that we support mixed-case column names on Oracle, I cannot find any tests with quotation marks in `db_column`.